microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -103,6 +103,15 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
 Collections aren't available when using **InPrivate** or **Guest** modes.
 
+<!-- ====================================================================== -->
+## Compose
+
+In the Bing side pane, Compose uses AI to write for you starting from a short prompt and "tone", "format", and "length" selections. Text generated by Compose can be copied or inserted into the text page directly. To access Compose, click on the blue Bing icon in the right hand pane > Compose (requires Bing waitlist approval). Text and selections inputed by the user is sent to an internal Microsoft Large Language Model (LLM).
+
+![Compose](./media/compose.png)
+
+The request is sent over a secure HTTPS connection with encryption and contains the input prompt, formatting selections, along with IDs associated with the user's Edge profile and device, to reference for subsequent requests. User input is not stored at all. 
```

Comment:
>  User input is not stored at all.

Leaving a note here as a reminder for me to ensure that we follow up on this.

Reply:
Following up with Eric Tang.

Reply:
@majenny maybe you can close this comment thread, b/c latest https://github.com/MicrosoftDocs/edge-developer/pull/2574/files no longer has "User input is not stored at all."; it has:
```
Requests for Compose are sent over a secure HTTPS connection. 
Each request contains the input prompt, formatting selections, 
and IDs that are associated with the user's Microsoft Edge profile and device, 
to reference for subsequent requests.
```